src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -19,21 +19,30 @@ import { ZoneClient2016 as Client } from "./classes/zoneclient";
 import { Vehicle2016 as Vehicle } from "./classes/vehicle";
 import { WorldObjectManager } from "./classes/worldobjectmanager";
 
-import { loadoutContainer, Weather2016 } from "../../types/zoneserver";
+import { inventoryItem, loadoutContainer, Weather2016 } from "../../types/zoneserver";
 import { h1z1PacketsType } from "../../types/packets";
 import { Character2016 as Character } from "./classes/character";
 import { H1Z1Protocol } from "../../protocols/h1z1protocol";
-import { _, initMongo, Int64String, isPosInRadius } from "../../utils/utils";
+import {
+  _,
+  initMongo,
+  Int64String,
+  isPosInRadius,
+  getDistance,
+  randomIntFromInterval,
+} from "../../utils/utils";
 
 import { Db, MongoClient } from "mongodb";
 import dynamicWeather from "./workers/dynamicWeather";
 
 // need to get 2016 lists
 const spawnLocations = require("../../../data/2016/zoneData/Z1_spawnLocations.json");
 const recipes = require("../../../data/2016/sampleData/recipes.json");
+const deprecatedDoors = require("../../../data/2016/sampleData/deprecatedDoors.json");
```

Comment:
what are them ?

Reply:
depreciated door placers that should be invisible but arent,